src/components/Editable.js

```diff
@@ -188,15 +166,65 @@ export const Editable = connect()(({ isEditing, thoughtsRanked, contextChain, sh
   }
 
   // using useRef hook to store throttled function so that it can persist even between component re-renders, so that throttle.flush method can be used properly
-  const throttledChangeRef = useRef(_.throttle(onChangeHandler, EDIT_THROTTLE, { leading: false }))
+  const throttledChangeRef = useRef(throttle(thoughtChangeHandler, EDIT_THROTTLE))
+
   ShortcutEmitter.on('shortcut', () => {
     throttledChangeRef.current.flush()
   })
 
-  useEffect(() => throttledChangeRef.current.flush, []) // clean up function when component unmounts (flushing throttle change)
```

Comment:
Editable doesn't re-render and doesn't `unmount` in its life cycle. Only time it unmounts is when it's parent thought is not rendered.  Flush on `onBlur` event is enough. Still we can keep the unmount flush. Should we do that ?

Reply:
Yes, let's keep it to be safe. I'm not sure `onBlur` will necessarily be called every time this component unmounts, and missing a flush would be disastrous.

Reply:
> @shresthabijay I'm getting a very strange error when I try to run `npm start`:
> 
> ```
> ./src/reducers/editingValue.js
>   Line 0:  Parsing error: ENOENT: no such file or directory, stat '/users/raine/projects/em/tsconfig.eslint.json'
> ```
> 
> It runs fine on the `dev` branch, and this PR is up-to-date with `dev`. The error itself is nonsensical since `/tsconfig.eslint.json` does exist. Not to mention that it doesn't seem to have anything to do with `./src/reducers/editingValue.js`.
> 
> Would be so grateful for you to investigate!

Hi Raine! It's a very strange issue. This PR is up-to-date with dev and has got all the changes related to typescript configuration. It works fine for me without any error. I tried looking for any possible ts configuration error but there are none. Have you tried updating the `typescript ` package to the latest version `^3.7.5` ?